noise_raster/noise_raster.py

```diff
@@ -297,7 +297,7 @@ def run(self):
 
                 # Vectorize energetically added raster including all noise sources
                 out_poly = self.dlg.mQgsFileWidget_out.filePath()
-                vectorize(out_final_ras, out_poly, selectedTableIndex)
+                vectorize(out_final_ras, out_poly, selectedTableIndex, temp_dir, logger)
 
                 # Reproject energetically added raster to EPSG:3035
                 reproject_3035(out_final_ras, out_ras)
```

Comment:
I looked into that, however there is too much additional logic in the reproject() function to make this possible. The reproject function contains additional logic to streamline data types and perform file conversions in preparation for merging. The reproject function also accepts a list as input, whereas the reproject_3035 accepts a single file and reprojects it to a static and known CRS. It would take some serious refactoring and the creation of additional functions to combine these two functions into one.